core/quantity/src/Formatter/Formatter.ts

```diff
@@ -350,13 +350,15 @@ export class Formatter {
       // we assume that stopping value is always positive
       posMagnitude = Math.floor(posMagnitude * precisionScale + FPV_ROUNDFACTOR) / precisionScale;
 
-      const denominator = (Math.pow(10, spec.format.stationOffsetSize!));
+      const baseFactor = spec.format.stationBaseFactor ?? 1;
+      const stationOffsetSize = spec.format.stationOffsetSize || 3; // Provide default if undefined
```

Comment:
Did you just add this default?  It appears to be new and undocumented.

The native code has a default value of 0 for station offset size.

The default should be consistent and documented.  In an ideal world we would have required a value if the type of format was station.  

I think a default of 0 is reasonable because it will incentivize people to actually explicitly set one.

Reply:
We validate that stationOffsetSize is a required field when deserializing from JsonProps, I wanted to get rid of this non null assertion. I changed the stationOffsetSize default to be 0 now, documented it in the doccomment for that field too